### ISSUE TITLE
Paths and how to work with it?

### ISSUE THREAD

**Issue by Gavolot** [johanhelsing/bevy_pkv#49]
It's not exactly a mistake. But I read the documentation and didn’t understand anything about how to work with them.

Your library is working. But I don’t understand how I can, for example, “delete” the database for tests.

Because now I can’t do this, because I don’t know where it is in Windows. I'm exclusively making a PC game.

**Comment by Gavolot** [comment]
It's important to clarify, I'm relatively new to Rust, but not new to programming.

**Comment by johanhelsing** [comment]
The point of this library is that you should not have to think about paths, it's just created in the right directory.

There is a clear method, does this do what you want to do?

On windows, the db is created somewhere inside appdata

**Comment by Gavolot** [comment]
yes, I wanted to simulate a situation where the “User” deleted the database or decided to transfer the “saves” to another computer

**Comment by Gavolot** [comment]
I found where it is by default in Windows. I had already forgotten that Windows has a “recently changed” filter and the ability to sort by date.

My last question is, can I somehow force the database to be saved in the game's .exe directory?
